plugins/scale/scale.cpp

```diff
@@ -817,7 +854,7 @@ class wayfire_scale : public wf::plugin_interface_t
         y = workarea.y + (int)spacing;
         height = (workarea.height - (lines + 1) * (int)spacing) / lines;
 
-        std::sort(views.begin(), views.end());
+        view_sort(views, grid_cols);
```

Comment:
Since grid_cols is a class member variable, if you make view_sort() a class member function, you don't have to pass grid_cols explicitly. This also makes the need for the misleadingly named `wcnt` variable to be tossed out.

```suggestion
        view_sort(views);
```

Reply:
done